test/thrift/parser/file_group_test.exs

```diff
@@ -0,0 +1,18 @@
+defmodule Thrift.Parser.FileGroupTest do
+  use ExUnit.Case
+  use ThriftTestHelpers
+
+  alias Thrift.Parser.FileGroup
+  alias Thrift.Parser.Models.Constant
+
+  test "constant module uses suitable existing name" do
+    with_thrift_files([
+      "myservice.thrift": """
+      const float PI = 3.14
+      service MyService {}
+      """, as: :file_group, parse: "myservice.thrift"]) do
+
+      assert :"Elixir.MyService" == FileGroup.dest_module(file_group, Constant)
```

Comment:
I'd like to see some stronger asserts here, for example that the constant is generated in the correct module.